### PR TITLE
Docs: Update installation page for package split

### PR DESCRIPTION
**Pull request by acozine**
##### SUMMARY
This PR adds information to the installation page about the difference between the Ansible community package and `ansible-core`, reorganizes the installation page a bit, and updates the section on running from source. 

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
docs.ansible.com

### CODE REVIEW

**Comment by acozine** [comment]
Related to #71883.

File: docs/docsite/rst/installation_guide/intro_installation.rst [hunk 1]
```diff
@@ -1,99 +1,192 @@
...
 
 .. _control_node_requirements:
 
 Control node requirements
-^^^^^^^^^^^^^^^^^^^^^^^^^
+-------------------------
 
-Currently Ansible can be run from any machine with Python 2 (version 2.7) or Python 3 (versions 3.5 and higher) installed. Ansible 2.11 will make Python 3.8 a soft dependency for the control node, but will function with the aforementioned requirements. Ansible 2.12 will require Python 3.8 or newer to function on the control node. Starting with Ansible 2.11, the project will only be packaged for Python 3.8 and newer.
+For your control node (the machine that runs Ansible), you can use any machine with Python 2 (version 2.7) or Python 3 (versions 3.5 and higher) installed. Ansible 2.11 will make Python 3.8 a soft dependency for the control node, but will function with the aforementioned requirements. Ansible 2.12 will require Python 3.8 or newer to function on the control node. Starting with Ansible 2.11, the project will only be packaged for Python 3.8 and newer.
```
Comment (felixfontein) [inline docs/docsite/rst/installation_guide/intro_installation.rst]:
```suggestion
For your control node (the machine that runs Ansible), you can use any machine with Python 2 (version 2.7) or Python 3 (versions 3.5 and higher) installed. ansible-core 2.11 and Ansible 4.0.0 will make Python 3.8 a soft dependency for the control node, but will function with the aforementioned requirements. ansible-core 2.12 and Ansible 5.0.0 will require Python 3.8 or newer to function on the control node. Starting with ansible-core 2.11, the project will only be packaged for Python 3.8 and newer.
```
Not sure if the last sentence also needs adjustment for Ansible CC @abadger.

File: docs/docsite/rst/installation_guide/intro_installation.rst [hunk 2]
```diff
@@ -1,99 +1,192 @@
...
-
+When choosing a control node, remember that any management system benefits from being run near the machines being managed. If you are using Ansible to manage machines in a cloud, consider using a machine inside that cloud as your control node. In most cases Ansible will perform better from a machine on the cloud than from a machine on the open Internet.
 
 .. warning::
 
     Ansible 2.11 will make Python 3.8 a soft dependency for the control node, but will function with the aforementioned requirements. Ansible 2.12 will require Python 3.8 or newer to function on the control node. Starting with Ansible 2.11, the project will only be packaged for Python 3.8 and newer.
 
-.. warning::
-
-    Please note that some modules and plugins have additional requirements. For modules these need to be satisfied on the 'target' machine (the managed node) and should be listed in the module specific docs.
```
Comment (felixfontein) [inline docs/docsite/rst/installation_guide/intro_installation.rst:42]: The part on plugins needs to stay here.

File: docs/docsite/rst/installation_guide/intro_installation.rst [hunk 3]
```diff
@@ -1,99 +1,192 @@
...
+
+Although you do not need a daemon on your managed nodes, you do need a way for Ansible to communicate with them. For most managed nodes, Ansible makes a connection over SSH and transfers modules using SFTP. If SSH works but SFTP is not available on some of your managed nodes, you can switch to SCP in :ref:`ansible.cfg <ansible_configuration_settings>`. For any machine or device that can run Python, you also need Python 2 (version 2.6 or later) or Python 3 (version 3.5 or later).
 
-On the managed nodes, you need a way to communicate, which is normally SSH. By
-default this uses SFTP. If that's not available, you can switch to SCP in
-:ref:`ansible.cfg <ansible_configuration_settings>`.  You also need Python 2 (version 2.6 or later) or Python 3 (version 3.5 or
-later).
+.. warning::
+
+    Please note that some modules and plugins have additional requirements. For modules these need to be satisfied on the 'target' machine (the managed node) and should be listed in the module specific docs.
```
Comment (felixfontein) [inline docs/docsite/rst/installation_guide/intro_installation.rst]:
```suggestion
    Please note that some modules have additional requirements. For modules these need to be satisfied on the 'target' machine (the managed node) and should be listed in the module specific docs.
```
Plugin dependencies are not of interest for target nodes.

File: docs/docsite/rst/installation_guide/intro_installation.rst [hunk 4]
```diff
@@ -1,99 +1,192 @@
...
+You can choose any of the following ways to install the Ansible community package:
 
 * Install the latest release with your OS package manager (for Red Hat Enterprise Linux (TM), CentOS, Fedora, Debian, or Ubuntu).
 * Install with ``pip`` (the Python package manager).
-* Install ``ansible-base`` from source to access the development (``devel``) version to develop or test the latest features.
+
+Installing `ansible-core`
+-------------------------
+
+Ansible also distributes a minimalist object called ``ansible-core`` (or ``ansible-base in version 2.10). It contains the Ansible language, runtime, and a short list of core modules and other plugins. You can build functionality on top of ``ansible-core`` by installing collections from Galaxy, Automation Hub, or any other source.
```
Comment (felixfontein) [inline docs/docsite/rst/installation_guide/intro_installation.rst]:
```suggestion
Ansible also distributes a minimalist object called ``ansible-core`` (or ``ansible-base`` in version 2.10). It contains the Ansible language, runtime, and a short list of core modules and other plugins. You can build functionality on top of ``ansible-core`` by installing collections from Galaxy, Automation Hub, or any other source.
```

File: docs/docsite/rst/installation_guide/intro_installation.rst [hunk 5]
```diff
@@ -118,13 +211,13 @@ On CentOS:
 
 RPMs for RHEL 7 and RHEL 8 are available from the `Ansible Engine repository <https://access.redhat.com/articles/3174981>`_.
 
-To enable the Ansible Engine repository for RHEL 8, run the following command:
+To enable the Ansible repository for RHEL 8, run the following command:
```
Comment (felixfontein) [inline docs/docsite/rst/installation_guide/intro_installation.rst]: shouldn't the Engine stay here (and below)?

Comment (acozine) [reply docs/docsite/rst/installation_guide/intro_installation.rst]: Ah, you're right. I was thinking about the GitHub repository, but this is referring to something different.

File: docs/docsite/rst/installation_guide/intro_installation.rst (hunk 2 above)
Comment (acozine) [reply docs/docsite/rst/installation_guide/intro_installation.rst:42]: I didn't delete it, I just moved it down.

File: docs/docsite/rst/installation_guide/intro_installation.rst (hunk 3 above)
Comment (acozine) [reply docs/docsite/rst/installation_guide/intro_installation.rst]: Ahhh, I see what you mean. Okay, let me update the control node section.

File: docs/docsite/rst/installation_guide/intro_installation.rst (hunk 2 above)
Comment (acozine) [reply docs/docsite/rst/installation_guide/intro_installation.rst:42]: Addressed in 8f0bcbfe4682691632489345d962aea9d74362a0

File: docs/docsite/rst/installation_guide/intro_installation.rst (hunk 3 above)
Comment (acozine) [reply docs/docsite/rst/installation_guide/intro_installation.rst]: Addressed in 8f0bcbfe4682691632489345d962aea9d74362a0

File: docs/docsite/rst/installation_guide/intro_installation.rst (hunk 5 above)
Comment (acozine) [reply docs/docsite/rst/installation_guide/intro_installation.rst]: Addressed in 28719122db32af5ccbbf9944cb1cf04ffc8f4d10

**Comment by lmalivert** [comment]
@acozine LGTM 💯

**Pull request merged**